src/main/java/com/datalogics/pdf/hsm/samples/HsmManagerFactory.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2016 Datalogics Inc.
+ */
+
+package com.datalogics.pdf.hsm.samples;
+
+/**
+ * Used to get instances of a HsmManager object.
+ */
+public final class HsmManagerFactory {
+
+    public static final int LUNA_SA_HSM = 1;
```

Comment:
I'm going to lean on _Effective Java_ and ask for an `enum` instead of constants here.